index.html

```diff
@@ -54,5 +54,134 @@ <h2 class="header-subtitle text-white font-inter">I’m a Software Developer</h2
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+              </p>
+              <ul class="tags">
+                <li class="tag text-white dark-tag mb-6">HTML</li>
+                <li class="tag text-white dark-tag mb-6">Bootstrap</li>
+                <li class="tag text-white dark-tag mb-6">Ruby</li>
+              </ul>
+            </div>
+          </div>
+          <button class="btn w-full">See Project</button>
+        </div>
+        <div class="project-card">
+          <div class="project">
+            <div>
+              <h3 class="project-title text-white font-crete-round">Profesional Art Printing Data</h3>
+              <p class="project-description text-white mt-12">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+              </p>
+              <ul class="tags">
+                <li class="tag text-white dark-tag mb-6">HTML</li>
+                <li class="tag text-white dark-tag mb-6">Bootstrap</li>
+                <li class="tag text-white dark-tag mb-6">Ruby</li>
+              </ul>
+            </div>
+          </div>
+          <button class="btn w-full">See Project</button>
+        </div>
+        <div class="project-card">
+          <div class="project">
+            <div>
+              <h3 class="project-title text-white font-crete-round">Profesional Art Printing Data</h3>
+              <p class="project-description text-white mt-12">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+              </p>
+              <ul class="tags">
+                <li class="tag text-white dark-tag mb-6">HTML</li>
+                <li class="tag text-white dark-tag mb-6">Bootstrap</li>
+                <li class="tag text-white dark-tag mb-6">Ruby</li>
+              </ul>
+            </div>
+          </div>
+          <button class="btn w-full">See Project</button>
+        </div>
+        <div class="project-card">
+          <div class="project">
+            <div>
+              <h3 class="project-title text-white font-crete-round">Profesional Art Printing Data</h3>
+              <p class="project-description text-white mt-12">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+              </p>
+              <ul class="tags">
+                <li class="tag text-white dark-tag mb-6">HTML</li>
+                <li class="tag text-white dark-tag mb-6">Bootstrap</li>
+                <li class="tag text-white dark-tag mb-6">Ruby</li>
+              </ul>
+            </div>
+          </div>
+          <button class="btn w-full">See Project</button>
+        </div>
+      </section>
+    </main>
```

Comment:
- [ ] You have done a truly amazing job, the best project I have seen so far! But the "About myself" section (placed after the projects and before the contact form) me section is missing, please make sure to add it into the project:
<img width="132" alt="Screenshot 2022-03-17 at 20 32 37" src="https://user-images.githubusercontent.com/77038610/158848661-d2a3cb4f-4c4c-4525-831d-3559111832c2.png">

[Link](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) to project requirements for this milestone

Reply:
Oh, I forget that I'll fix it soon.